bin/kn-init

```diff
@@ -9,6 +9,7 @@ echo "Initializing $INIT_DIR deployment directory for cloud provider $host_cloud
 
 # Copy config, scripts and template files to init dir
 cp "/opt/KubeNow/templates/config.tfvars.$host_cloud-template" "/KubeNow_root/config.tfvars"
+cp "/opt/KubeNow/templates/inventory"*"-template" "/KubeNow_root/"
```

Comment:
Why do we need a `*`?

Reply:
I guess it is because in the openshift project I needed multiple bootstrap inventory templates. (* has to be outside "" for filename expansion)

Reply:
This could cause side effects, maybe you can have this in your OpenShift plugin? If difficult we can keep it as it is otherwise!

Reply:
Let's keep it for know, and then maybe it disappears when the plugin-thing gets more developed:-)